sdk/turing/router/config/router_config.py

```diff
@@ -250,7 +271,7 @@ def to_open_api(self) -> OpenApiModel:
         )
 
     def _get_default_route_id(self):
-        default_route_id = self.default_route_id
+        default_route_id = None
```

Comment:
By setting `None` here, we will be clearing this value for routers with custom ensemblers.